README.md

```diff
@@ -59,25 +59,24 @@ If you choose not to use FastLane, this can be done manually through the followi
 generated `mapping.txt` file manually before any stack traces received from that version of your app can be deobfuscated.
 
 1. Install Flurry SDK 6.7.0 or greater.
-1. Apply the Flurry android crash plugin to your app's build
+1. Add the Flurry android crash plugin to your project's build.gradle file.
    ```
      buildscript {
       repositories {
        maven {
         url "https://plugins.gradle.org/m2/"
        }
-       maven {
-        url  "http://yahoo.bintray.com/maven"
-       }
       }
       dependencies {
-       classpath "gradle.plugin.com.flurry:symbol-upload:1.2.0"
+       classpath "gradle.plugin.com.flurry:symbol-upload:4.2.1"
```

Comment:
Thanks for updating these instructions.

This should change if you update the version (see below)...

Reply:
updated plugin version